### ISSUE TITLE
Notifications screen- Intro

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#317]
On notifications screen, while using accessibility controls

When swiping from left to right, the focus goes on to the three dots even though they are not visible. (should be removed)

For the Go back button, invisible area should be added to make it easy to use. the minimum area is 44px by 44px.

**Comment by Rajat-Sen** [comment]
@Jiandong-Guo, please remove the dots completely instead of hiding. Thanks!

**Comment by Rajat-Sen** [comment]
Update: Still hasn't been fixed.

**Comment by Jiandong-Guo** [comment]
There is no hidden thing for three dots,  The label was fixed, Test it in next release

**Comment by AJBouzidi** [comment]
@ForumG-SC is this been tested?

**Comment by ForumG-SC** [comment]
> @ForumG-SC is this been tested?

I don't think Rajat has tested this as he has not closed the ticket.

**Comment by Rajat-Sen** [comment]
It's been fixed!
